lnclient/lnd/lnd.go

```diff
@@ -1197,7 +1211,6 @@ func (svc *LNDService) GetPubkey() string {
 func fetchNodeInfo(ctx context.Context, client *wrapper.LNDWrapper) (*lnclient.NodeInfo, error) {
 	resp, err := client.GetInfo(ctx, &lnrpc.GetInfoRequest{})
 	if err != nil {
-		logger.Logger.WithError(err).Error("Failed to fetch node info")
```

Comment:
why remove the error?

Reply:
I think it's ok to add it (we are logging at a different function within the LNClient and it's easier to debug)

Reply:
Done